tests/regression/test_hypre_ams.py

```diff
@@ -96,12 +94,45 @@ def test_homogeneous_field_nonlinear():
               'ksp_type': 'cg',
               'ksp_itmax': '30',
               'ksp_rtol': '1e-15',
-              'ksp_atol': '1e-15',
               'pc_type': 'python',
               'pc_python_type': 'firedrake.HypreAMS',
-              'pc_hypre_ams_zero_beta_poisson': True,
+              'pc_hypre_ams_zero_beta_poisson': True
               }
 
     solve(a - L == 0, u, bc, solver_parameters=params)
     B = project(curl(u), V0)
     assert numpy.allclose(B.dat.data_ro, numpy.array((0., 0., 1.)), atol=1e-6)
+
+@pytest.mark.skipcomplex
```

Comment:
Sorry, this needs an extra blank line to pass linting (you can check these errors locally with `make lint` in the firedrake src directory).